src/lv_misc/lv_area.c

```diff
@@ -208,3 +257,24 @@ void lv_area_increment(lv_area_t * a_p, const lv_coord_t amount)
 /**********************
  *   STATIC FUNCTIONS
  **********************/
+
+static bool lv_point_within_circle(const lv_area_t * area, const lv_point_t * p)
+{
+    lv_coord_t r = (area->x2 - area->x1) / 2;
+
+    /* Ellipse center */
```

Comment:
/* Ellipse center */
 
```suggestion
    /* Circle center */
```

Reply:
I actually had it in mind to change that before committing, and then I completely forgot! Thanks!